### PR TITLE
updated release schedule for 17

### PR DESCRIPTION
**Pull request by wiswedel**

### CODE REVIEW

File: admin_manual/installation/source_installation.rst [hunk 1]
```diff
@@ -100,7 +100,7 @@ If you get a result, the module is present.
 
 Required:
 
-* PHP (>= 7.0, 7.1 or 7.2)
+* PHP (7.0, 7.1, 7.2 or 7.3)
```
Comment (nickvergessen) [inline admin_manual/installation/source_installation.rst]:
```suggestion
* PHP (7.1, 7.2 or 7.3)
```

Comment (nickvergessen) [reply admin_manual/installation/source_installation.rst]: 7.0 is not supported in master

File: admin_manual/release_schedule.rst [hunk 2]
```diff
@@ -7,6 +7,8 @@ This page gives an overview of the currently supported and released versions as
 +--------+-------------------+------------------+--------------------------+---------------------------+
 |        | release date      | end of life      | latest version           | next version              |
 +========+===================+==================+==========================+===========================+
+| **17** | 2019-08-08        | 2020-08          |                       -  | *17.0.0 (2019-08-08)*     |
++--------+-------------------+------------------+--------------------------+---------------------------+
```
Comment (MorrisJobke) [inline admin_manual/release_schedule.rst]: This was on purpose - we don't want to put major release dates into the manual, as this is a document that is also shipped in the release and thus easily out of date.

Comment (wiswedel) [reply admin_manual/release_schedule.rst]: > This was on purpose - we don't want to put major release dates into the manual, as this is a document that is also shipped in the release and thus easily out of date.

Ah ok. didn't know that.

**Pull request merged**